third_party/nvfuser/csrc/transform_iter.cpp

```diff
@@ -172,6 +172,38 @@ void ReplayTransformations::handle(Swizzle2D* swizzle_2d) {
   id_map_[swizzle_2d->outY()] = outs.second;
 }
 
+void ReplayTransformations::handle(Resize* exp) {
+  auto id_in = exp->in();
+
+  auto it = id_map_.find(id_in);
+  if (it == id_map_.end()) {
+    if (error_on_failure_) {
+      TORCH_INTERNAL_ASSERT(
+          false, "Transform traversal failed, dependencies not met.");
+    } else {
+      return;
+    }
+  }
+
+  auto mapped = (*it).second;
```

Comment:
nit:
```suggestion
  auto mapped = it->second;
```

Reply:
Done. Also changed the same patterns in this file.